src/server/common.cc

```diff
@@ -453,7 +455,8 @@ RandomPick UniquePicksGenerator::Generate() {
   return max_index;
 }
 
-ThreadLocalMutex::ThreadLocalMutex() {
+ThreadLocalMutex::ThreadLocalMutex()
+    : big_value_enabled_(absl::GetFlag(FLAGS_serialization_max_chunk_size) != 0) {
```

Comment:
I do not want us to access absl::GetFlag here. Instead, create a global variable in common.cc like `max_memory_limit` and initialize it inside `Service::Init`